src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,120 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.exception.RegistrationException;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private static final String DEFAULT_PASSWORD = "123456";
+    private static final String DEFAULT_LOGIN = "123456";
+    private static final String SOME_LOGIN = "some login";
+    private static final String USER1_LOGIN = "user1Login";
+    private static final String USER2_LOGIN = "user2Login";
+    private static final String SHORT_LOGIN = "short";
+    private static final Integer DEFAULT_AGE = 25;
+    private static final int UNCORRECT_AGE = 4;
+    private static RegistrationServiceImpl registrationService;
+    private final User user = new User();
+
+    @BeforeAll
+    static void beforeAll() {
+        registrationService = new RegistrationServiceImpl();
+    }
```

Comment:
I don't think there is a necessity of using `@BeforeAll` with static variables. In addition - you don't need to declare some generic `User`, since you should prepare those users per each test separately - you're testing `RegistrationService` here

why not just:
```suggestion
    private final RegistrationServiceImpl registrationService = new RegistrationServiceImpl();
```

**Pay attention, that you will have to change all the tests, that were referring to this generic user**

Reply:
But in [this](https://github.com/mate-academy/jv-registration-validation/pull/722#discussion_r1314888976) comment Andrii said that i need to do this